app/models.py

```diff
@@ -30,6 +32,8 @@ class Service(db.Model):
                                nullable=False)
     data = db.Column(JSON)
 
+    status = db.Column(db.String, index=False, unique=False, nullable=False)
```

Comment:
It's not so much the query speed that I'm worried about with the string columns, but rather the validation/flexibility implications. E.g. what if we decide to rename "published" to something else.